neqo-transport/src/send_stream.rs

```diff
@@ -1030,7 +1030,7 @@ mod tests {
         flow_mgr.borrow_mut().conn_increase_max_credit(2);
         let conn_events = ConnectionEvents::default();
 
-        let _s = SendStream::new(4.into(), 100, flow_mgr.clone(), conn_events.clone());
+        let _s = SendStream::new(4.into(), 100, flow_mgr, conn_events.clone());
```

Comment:
```suggestion
        let _s = SendStream::new(4.into(), 100, flow_mgr, Rc::clone(&conn_events));
```

Reply:
I think we need to keep conn_events.clone() since ConnectionEvents is Clone, but it's not an Rc.

Reply:
Of course, I misremembered.